docs/HOW_TO_UPGRADE.md

```diff
@@ -84,6 +84,18 @@ These are custom modules and this is what you have to keep in mind when updating
         * ES: "Asamblea" for "Consejo de participación"
 
       Modified files are:
+#### Temporal fix: format meetings's start_time / end_time in meeting_form.rb map_model method
+
+Currently, in the file:
+- app/forms/decorators/decidim/meetings/admin/meeting_form_decorator.rb
+we have overrided `def map_model(model)` method to format start_time and end_time fields
+
+The reason for this, is that a wrong format arrived to from_params method on Rectify form
+This happened only when after having created a meeting with start_time / end_time, we wanted to edit it and change only one date field, this is only start_time or end_time.
+In that moment, de form date's validation fields threw an error because the other date field was in wrong format, so form_builder wasn't able to assign to form.
+
+In next versions, this issue will be patched in `decidim/decidim`, so this override could be removed:
```

Comment:
do we know exactly at which version of decidim the issue was fixed?

Reply:
we don't know, the issue is the same that https://github.com/gencat/participa/pull/165, and it has no specified an issue of decidim or a PR

Reply:
I'll check with @Leusev